runtime/tools/python/ttrt/common/run.py

```diff
@@ -117,7 +117,7 @@ def initialize_api():
             help="pickup the kernels from disk (/tmp) instead of the flatbuffer",
         )
         Run.register_arg(
-            name="--enable-async-ttnn",
+            name="--disable-async-ttnn",
```

Comment:
@tapspatel currently in runtime we disable async ttnn by default, so this should be --enable-async-ttnn (as well as in the other tests modified)

Reply:
ah I see, is it okay if I change the default to True then? I wanted to keep it consistent with the other flag
eg
--disable-ignore-tile-shape
--disable-empty-op-row-major
etc

Reply:
Ah good catch, I was going to ask what motivated this change since it's wrong now (disable flag drives enable field). Don't think we want to change defaults since it affects runs outsied of ttrt, right?

Reply:
How would the user enable it then?

Reply:
Lemme just revert it back to what it was, seems like it causes too many problems then it fixes